topics/css/README.md

```diff
@@ -1,5 +1,14 @@
 # CSS
 
+Hojas de estilo en cascada, o CSS por su sigla en inglés
+(Cascading Style Sheets) es la tecnología utilizada para
```

Comment:
```suggestion
(Cascading Style Sheets), es una tecnología utilizada para
```

Me suena más preciso decir que CSS es _una_ tecnología para dar estilos, y no implicar que es la única, también podemos usar atributos de HTML y JavaScript (CSS in JS, etc) como estrategias y tecnologías alternativas a CSS (con sus pros y contras)... :thinking: